agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3760,34 +3687,39 @@ bool slave_thread::hostap_services_on()
     return success;
 }
 
-void slave_thread::monitor_stop()
+void slave_thread::fronthaul_stop()
 {
-    bool did_stop = false;
+    LOG(INFO) << "fronthaul stop";
+
     if (monitor_socket) {
         remove_socket(monitor_socket);
         delete monitor_socket;
         monitor_socket = nullptr;
-        did_stop       = true;
     }
 
-    // kill monitor slave pid
-    os_utils::kill_pid(config.temp_path, std::string(BEEROCKS_MONITOR) + "_" + config.hostap_iface);
+    if (ap_manager_socket) {
+        remove_socket(ap_manager_socket);
+        delete ap_manager_socket;
+        ap_manager_socket = nullptr;
+    }
 
-    if (did_stop)
-        LOG(DEBUG) << "monitor_stop() - done";
+    // kill monitor slave pid
+    os_utils::kill_pid(config.temp_path,
```

Comment:
Isn't the monitor is just a thread in our process now? What are we trying to kill here?

Reply:
It kills the fronthaul process. The comments are wrong. Fix them.